docs/gambit/mutation-verifier.md

```diff
@@ -36,96 +36,80 @@ Once you have updated your `certora-cli` installation using `pip` to get the rel
+| `--prover_version`             | specify the version of `certoraRun` to use for verification. Defaults to the latest installed version                 |
+| `--server`                     | specify the server environment to run on. Defaults to the environment specified in `prover.conf`                    |
+| `--debug`                      | show additional logging information during execution                                                                  |
+| `--gambit_out`                 | specify the output directory for gambit . Defaults to a new directory is added in the working directory               |
+| `--applied_mutants_dir`        | specify the target directory for mutant verification build files. Defaults to a special directory in Prover internals |
+| `--ui_out`                     | specify the directory of the mutant verification report JSON used for the web UI                                      |
+| `--dump_link`                  | specify a text file to write the UI report link                                                                       |
+| `--collect_file`               | specify the collect file from which to run in asynchronous mode                                                       |
+| `--sync`                       | enable synchronous execution                                                                                          |
+| `--max_timeout_attempts_count` | specify the maximum number of times a web request is attempted                                                        |
+| `--request_timeout`            | specify the length in seconds for a web request timeout                                                               |
+| `--poll_timeout`               | specify the number of minutes to continue polling a submitted task in sync mode                                       |
+
+## Troubleshooting
 
 At the moment, there are a few ways in which `certoraMutate` can fail. Here are some suggestions on how to troubleshoot when that happens. We are actively working on mitigating them.
 
-- Make sure the run script you use to run the Prover does not have the `--send_only` flag or any commented out (using `#`) lines.
-- Since Gambit requires you to provide the solidity compiler flags to compile the mutants, sometimes it might be useful to first identify what those flags should be. See {ref}`gambit-config` for more information. A strategy you can adopt is
-  * first run `gambit` without going through `certoraMutate` (you likely have either `gambit-linux` or `gambit-macos` binaries in your path already if you are running the tool).
-  * Make a `foo.json` file and copy the content of the `"gambit":` field in it.
-  * Run `gambit-OS mutate --json foo.json` to identify the issue.
+- Sometimes it might be useful to first run `gambit` without going through `certoraMutate`
+  (you likely have the `gambit` binary in your path already if you are running the tool).
```

Comment:
Probably not true after the recent changes. Maybe we should add a gambit access command, or somehow help update the PATH variable